### DISCUSSION TITLE
Student developer pack application rejected even if I have the correct documents

### DISCUSSION THREAD

**Discussion by laxiee** [community/community#124836] [GitHub Education]
### Hi everyone,

I am trying to apply for a student developer pack but I am not able to get selected in the application even if I have the correct documents to verify my education. I have a school id card which is dated for the session 2024-2025. It also has my name and picture. What else am I lacking?
![Document_2024-05-08_215923ba.jpg](https://github.zendesk.com/attachments/token/1plhgkyArb3QDOS6qexgGDCet/?name=Document_2024-05-08_215923ba.jpg)

**Comment by davevad93** [comment]
Hi @laxiee ,

If your application has been rejected try to upload another document with a current date that shows that you are currently studying there (for example a photo of your last exam results or another document that shows a date). It's possible that your application has been rejected because your ID shows 2 dates, one that is the academic year (2024-2025) and the other one on your photo (28-04-2023). I say this because I think it's a bot that processes all applications and it's possible that the date taken into account is that one, which would explain the reason for the rejection as it wouldn't prove your current enrollment at your school. Last year when I made my request I uploaded a photo of my enrollment sheet, it was rejected because even though it showed the academic year and the school's name, for GitHub it wasn't enough proof to verify my student status so I uploaded a photo of my last exams showing a date with a DD/MM/YYYY format and finally my request was accepted.
 I advise you to do the same, if that doesn't work either, open a ticket on [GitHub Support](https://support.github.com/request/education).

**Reply by laxiee** [reply]
I uploaded the picture attached below but still wasn't able to get accepted. It contains an exact date in the dd/mm/yyyy format till when my education is valid in the school. It doesnt contain any other YYYY. what options do I have now. When I try to raise a ticket it shows -  Individual manual reviews and status updates are no longer provided. 

![image](https://github.com/community/community/assets/82639731/4a1789c9-ffad-49db-914a-1934356a5d9c)

**Reply by davevad93** [reply]
Hey @laxiee , what are the reasons for the rejection?

**Reply by laxiee** [reply]
We need more information demonstrating your academic affliation.
